packages/discovery/src/discovery/config/RawDiscoveryConfig.ts

```diff
@@ -12,7 +8,7 @@ export type DiscoveryContract = z.infer<typeof DiscoveryContract>
 export const DiscoveryContract = z.object({
   ignoreDiscovery: z.optional(z.boolean()),
   proxyType: z.optional(ManualProxyType),
-  ignoreInWatchMode: z.optional(z.array(z.string())),
+  ignoreInWatchMode: z.z.optional(z.array(z.string())),
```

Comment:
Does it provide some form of stronger validation - the double z? 😄

Reply:
I wonder why does TS not shout about it

Reply:
7948981e